docs/docs/03-light-clients/05-tendermint/01-overview.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Overview
+sidebar_label: Overview
+sidebar_position: 1
+slug: /ibc/light-clients/tendermint/overview
+---
+
+# `07-tendermint`
+
+## Overview
+
+The tendermint client is the first and most deployed light client in IBC. It implements the IBC Light Client interface to track a counterparty running Comet BFT consensus. Note: Tendermint is the old name of Comet BFT which has been retained in IBC to avoid expensive migration costs.
```

Comment:
no harm to throw in a link here to the cometbft repo

```suggestion
The tendermint client is the first and most deployed light client in IBC. It implements the IBC Light Client interface to track a counterparty running [CometBFT](https://github.com/cometbft/cometbft) consensus. Note: Tendermint is the old name of CometBFT which has been retained in IBC to avoid expensive migration costs.
```

sidenote, I think they still refer to cometbft as a implementation running "Tendermint consensus", it might be good to clarify that before we merge this